backend/apps/owasp/admin/mixins.py

```
@@ -12,7 +12,13 @@
 
 
 class BaseOwaspAdminMixin:
-    """Base mixin for OWASP admin classes providing common patterns."""
+    """
+    Base mixin for OWASP admin classes.
+
+    Provides common configuration patterns—such as default list_display,
+    list_filter, and search_fields—so individual ModelAdmin classes can avoid
+    duplicated boilerplate.
+    """
```

Ruff (0.14.5)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)